docs/sources/installation/upgrading.md

```diff
@@ -309,4 +309,8 @@ The database table _temp\_user_, that tracks user invites, is subject to a datab
 The database table _dashboard\_snapshot_, that stores dashboard snapshots, adds a new column _dashboard\_encrypted_ for storing an encrypted snapshot. 
 NOTE: Only snapshots created on Grafana 7.3 or later will use this column to store snapshot data as encrypted. Snapshots created before this version will be unaffected and remain unencrypted.
 
+### Use of the root group in the Docker images
+
+The Grafana Docker images use the `root` group instead of the `grafana` group. This change can cause builds to break for users who extend the Grafana Docker image. Learn more about this change in the  [docker migration instructions]({{< relref "docker/#migrate-to-v73-or-later">}})
```

Comment:
Please change [docker migration instructions] to [Docker migration instructions]

Reply:
Fixed in separate pr as this one is already merged.